src/CalculatorService.js

```diff
@@ -0,0 +1,31 @@
+import Validator from './Validator.js';
+import {
+  sum,
+  getCustomSeparator,
+  getRemovedCustomSeparator,
+  getSplitedBySeparator,
+} from './util/inputUitl.js';
+
+export default class CalculatorService {
+  static sumUserInput(input, customSeparator) {
+    const splitedUserInput = getSplitedBySeparator(input, customSeparator);
+    return sum(splitedUserInput);
+  }
+
+  static validateUserInput(input) {
+    Validator.validateStartsWith(input);
+    const { customSeparator, processedUserInput } = this.parseUserInput(input);
+    Validator.validateUsedSeparator(processedUserInput, customSeparator);
+  }
+
+  static parseUserInput(input) {
+    if (input.startsWith('//')) {
+      Validator.validateCustomSeparator(input);
+      const customSeparator = getCustomSeparator(input);
+      const processedUserInput = getRemovedCustomSeparator(input);
+      return { customSeparator, processedUserInput };
+    }
+
+    return { customSeparator: null, processedUserInput: input };
+  }
```

Comment:
```suggestion
    if (input.startsWith('//')) {
      return { customSeparator: null, processedUserInput: input };
    }
    Validator.validateCustomSeparator(input);
    const customSeparator = getCustomSeparator(input);
    const processedUserInput = getRemovedCustomSeparator(input);
    return { customSeparator, processedUserInput };
  }
```

이런식으로 얼리리턴을 이용하시면 deps를 좀더 빼내실 수 있을거에요!:)

Reply:
헉..! 맞네요! 생각하지 못한 부분이였는데 감사합니다!!